README.md

```diff
@@ -173,6 +173,10 @@ print(str(r))
 
 The `Dockerfile` present in this repository and its respective `twilio/twilio-python` Docker image are currently used by Twilio for testing purposes only.
 
+### Using a Custom HTTP Client
```

Comment:
Put this above the Docker stuff. Docker is less important since it's just for our use.